gui/options.cpp

```diff
@@ -1229,6 +1231,25 @@ GlobalOptionsDialog::GlobalOptionsDialog()
 
 #endif // USE_TRANSLATION
 
+#ifdef USE_UPDATES
+	_updatesPopUpDesc = new StaticTextWidget(tab, "GlobalOptions_Misc.UpdatesPopupDesc", _("Update check:"), _("How often to check ScummVM updates"));
+	_updatesPopUp = new PopUpWidget(tab, "GlobalOptions_Misc.UpdatesPopup");
+
+	const int *vals = Common::UpdateManager::getUpdateIntervals();
+
+	while (*vals != -1) {
+		_updatesPopUp->appendEntry(Common::UpdateManager::updateIntervalToString(*vals), *vals);
+		vals++;
+	}
+
+	if (ConfMan.hasKey("updates_check"))
+		_updatesPopUp->setSelectedTag(ConfMan.getInt("updates_check"));
```

Comment:
``setSelectedTag`` will select nothing in case the value from the config file is not one of our predefined values. It looks like this case needs to be handled specially. I can't really tell from our sources what the implications here are, but it might be that ``getSelectedTag`` will return ``0xFFFFFFFF`` in this case. And I have no idea how this affects the GUI logic from a quick glance, but it looks like no precautions are made here?